templates/ra_guac_autoscale_public_alb.template.cfn.yaml

```
@@ -89,6 +93,18 @@ Parameters:
     Default: ''
     Description: Text/Label to display branding for the Guac Login page
     Type: String
+  CloudWatchAgentUrl:
+    AllowedPattern: '^$|^s3://.*\.rpm$'
+    Default: ''
+    Description: >-
+      (Optional) S3 URL to CloudWatch Agent installer. Example(CentOS):
```

The rpm is common to any EL-derivative, not just CentOS, and this whole template is dependent on being an EL-derivative...

```suggestion
      (Optional) S3 URL to CloudWatch Agent installer. Example:
```